.releaserc

```diff
@@ -1,12 +1,12 @@
 {
-  "branch": "master",
-  "tagFormat": "master+v${version}",
+  "branch": "release/1.12",
```

Comment:
I wonder if we could have this as ENV vars in Jenkinsfile it's just tooooooooooo many places to look at.

Reply:
its a JSON file :(

Reply:
Also I try to consolidate all information in that very file to have less places to look at 👍

Reply:
We could switch to a `js` file https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/configuration.md#configuration-file

Reply:
uh, nice! but lets not do it in this PR please.

Reply:
deal